### ISSUE TITLE
Simple project crashing with Python3.7.0

### ISSUE THREAD

**Issue by illera88** [marcelotduarte/cx_Freeze#399]
Is Python 3.7.0 supported? I just created a simple project and it works with Python 3.5.4 but it does not in Python 3.7.0 where it shows:

```
Fatal Python error: initfsencoding: unable to load the file system codec
zipimport.ZipImportError: can't find module 'encodings'
```

Thank you

**Comment by baneeishaque** [comment]
Same here.

**Comment by alec0xFFFFFF** [comment]
I got the same issue with `invalid flags 1530086518 in encodings` when running in Python 3.7.0
 Had to pip install cx_Freeze from a .whl instead of just calling pip install cx_Freeze as it would fail with  `/cxfreeze-postinstall does not exist` 
I pivoted back to my Python 3.6.5 environment and had no issues with pip installing it and then had no issues booting the converted executable.

**Comment by HotelCalifornia** [comment]
I'm getting a similar error:

```
Fatal Python error: initfsencoding: unable to load the file system codec
ImportError: invalid flags 1530079561 in 'encodings'

Current thread 0x00007fff8b0b7380 (most recent call first):
Abort trap: 6
```

~Going to try using python 3.6 again to see if it was the upgrade that broke it~ edit: #392 makes me think this is something else (or was potentially introduced in a minor update on 3.6)

**Comment by illera88** [comment]
@anthony-tuininga can you take a look at it? I think is related with the generated zip file which does not get properly parsed and therefore the `encodings` module does not get loaded producing the error.

Cheers

**Comment by MusicJelly** [comment]
@HotelCalifornia   same error, if you solve this problem, let me know. Thanks

**Comment by fazalmajid** [comment]
FWIW, I was encountering this error because of a stray environment variable:

    PYLIB=/Library/Frameworks/Python.framework/Versions/Current/lib/python2.3

removing it fixed the error.

**Comment by totaam** [comment]
> Is Python 3.7.0 supported?

Not without this fix:
https://github.com/anthony-tuininga/cx_Freeze/pull/395

**Comment by cocoliliace** [comment]
@MusicJelly I have the same error too

**Comment by HotelCalifornia** [comment]
sorry for the late reply. I've confirmed that builds work with python 3.6 (I haven't tried out the fix @totaam mentioned though)

**Comment by ghost** [comment]
I also have this problem in 3.7

**Comment by Ae-Mc** [comment]
I have this problem too

**Comment by Nikobraz** [comment]
I have the same problem

**Comment by Jerakin** [comment]
I get the same error.

Opening the app built with `python setup.py bdist_dmg` gives me the error
`LSOpenURLsWithRole() failed with error -10810 for the file /tool.app.`

Running the excecutable in the .app gives me 

```
Fatal Python error: Py_Initialize: unable to load the file system codec
ImportError: No module named 'encodings'

Current thread 0x00007fff9d7c2380 (most recent call first):
Abort trap: 6
```

Tried again today (different project) and get

```
Fatal Python error: initfsencoding: unable to load the file system codec
ImportError: invalid flags 1542114174 in 'encodings'

Current thread 0x00007ffff3fc83c0 (most recent call first):
Abort trap: 6
```


 tried with both `cx_Freeze 6.0b1` and `cx_Freeze 5.1.1` with python versions
```
python3.5.0
python3.6.6
python3.7.0
```

Tried including encodings in the setup.py script but it doesn't help
`buildOptions = dict(packages=[], excludes=[], includes=["encodings"])`

Python is running from `pyenv virtualenv`

**Comment by mobilinkd** [comment]
I am having the same problem.  I have tried both 5.1.1 and 6.0b1.  OS X build of a Glade project using gi. 
```
Fatal Python error: initfsencoding: unable to load the file system codec
ImportError: invalid flags 1540016716 in 'encodings'
```

**Comment by ajmontero** [comment]
I pulled down master and installed from source, which contains the fix for #395 I still get an error

> Fatal Python error: initfsencoding: unable to load the file system codec
ImportError: invalid flags 1530072455 in 'encodings'

> Current thread 0x000043d0 (most recent call first):

**Comment by elguavas** [comment]
i am getting this same problem still under python 3.7.2 with cx_freeze 5.1.1 installed via pip...

is there any workaround??

**Comment by elguavas** [comment]
ok i just found this patch elsewhere:
https://github.com/msys2/MINGW-packages/commit/4c18633ba2331d980f00aff075f17135399c43c5
making the changes to freeze.py in my install of 5.1.1 (64bit) fixes the problem for me!!

can somebody look at this and do a merge? please?

tia ;)

**Comment by grandec** [comment]
> ok i just found this patch elsewhere:
> [msys2/MINGW-packages@4c18633](https://github.com/msys2/MINGW-packages/commit/4c18633ba2331d980f00aff075f17135399c43c5)
> making the changes to freeze.py in my install of 5.1.1 (64bit) fixes the problem for me!!
> 
> can somebody look at this and do a merge? please?
> 
> tia ;)

Thanks, it worked for me as well.

**Comment by anthony-tuininga** [comment]
cx_Freeze 6 has just been released with support for Python 3.7. Thanks everyone for your huge quantities of patience!

**Comment by kk21986** [comment]
@anthony-tuininga I just installed 6.0 but still facing the same problem with python 3.7.1.
```
Fatal Python error: initfsencoding: Unable to get the locale encoding
ModuleNotFoundError: No module named 'encodings'

Current thread 0x00007f7a7eee1740 (most recent call first):
Aborted
```
